front/components/DataSourceResourceSelectorTree.tsx

```diff
@@ -14,33 +11,32 @@ import type {
 } from "@dust-tt/types";
 import type { ConnectorPermission, ContentNodeType } from "@dust-tt/types";
 import { CircleStackIcon, FolderIcon } from "@heroicons/react/20/solid";
-import { useCallback, useState } from "react";
+import { useEffect, useState } from "react";
 
 import { useConnectorPermissions } from "@app/lib/swr";
-import { classNames } from "@app/lib/utils";
 
 export default function DataSourceResourceSelectorTree({
```

Comment:
Do we still need this layer? Can't we simply use the Sparkle component in all places?